cli/dcoscli/package/main.py

```diff
@@ -682,7 +682,14 @@ def _bundle_uris(uris_directory, zip_file):
     :rtype: None
     """
 
-    for filename in sorted(os.listdir(uris_directory)):
+    uris = sorted(os.listdir(uris_directory))
+    # these uris will be found through a mustache template from the filename
+    # (without the extension) so make sure each name is unique.
+    uri_names = set(name.split('.')[0] for name in uris)
+    if len(uris) != len(uri_names):
+        raise DCOSException('Error bundling package. Multiple assets have the '
+                            'same filename {}'.format(uris))
```

Comment:
Let's change the wording a little: "Error bundling package. Multiple assets map to the same property name (periods [.] are replaced with dash [-]): ..."

What do you think?